docs/manual-indexes.adoc

```diff
@@ -0,0 +1,166 @@
+= Manual Indexes
+
+== Data Used
+
+In the below examples I used the flight data setup from the following link:
+
+https://github.com/nicolewhite/neo4j-flights
+
+
+== Enable Manual Indexes on Node and Relationship Property
+
+Below are the steps to setup manual index on the nodes and relationship.
+In order to do so, one needs to first enable auto index and identify the property on the nodes and relationships
+that will be indexed.
+Once the property is identified set enable the auto index and set the property as shown in the configuration file.
+Below is the configurations setting to enable auto index in versions 3.0 and above.
+Here we are adding manual index on property `taxi_time` on Relationship `DESTINATION` and on property `flight_num` on Node with label `Flight`.
+
+
+In conf/neo4j.conf file
+
+----
+dbms.auto_index.relationships.enabled=true
+dbms.auto_index.relationships.keys=taxi_time, flight_num
+----
+
+Restart Neo4j.
+
+== Adding Manual Index on Relationship Property
+
+Once the auto indexes are enabled.
+
+One will have to add the relationship property to the index.
+This can be achieved using the APOC stored procedure - `apoc.index.addRelationship`.
+
+----
+match (f:Flight)-[r:DESTINATION]->(a:Airport)
+with r
+call apoc.index.addRelationship(r,["taxi_time"])
```

Comment:
also this works differently than the auto-index b/c the index-name is the relationship-type and here you only add one property whereas in the auto-index you had two rel-properties indexed